lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/ComponentViewController.java

```diff
@@ -114,7 +116,11 @@ public void applyBottomInset() {
 
     @Override
     public int getBottomInset() {
-        int navigationBarInset = resolveCurrentOptions().statusBar.isHiddenOrDrawBehind() ? 0 : NavigationBarUtils.getNavigationBarHeight(getActivity());
+        Context activity = getActivity();
+        boolean isPortrait = activity.getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT;
+        boolean shouldSetBottomInset = !(resolveCurrentOptions().bottomTabsOptions.isHiddenOrDrawBehind() || resolveCurrentOptions().statusBar.isHiddenOrDrawBehind());
+        int navigationHeight = NavigationBarUtils.getNavigationBarHeight(activity);
+        int navigationBarInset = !shouldSetBottomInset && isPortrait ? 0 : navigationHeight;
```

Comment:
tested on android tablet w/ gesture navigation?

Reply:
This is one of those changes that we need to test across lots of different android devices